cedar-policy-validator/src/human_schema/err.rs

```diff
@@ -16,6 +16,8 @@ pub enum UserError {
     EmptyList,
     #[error("Invalid escape codes")]
     StringEscape(NonEmpty<UnescapeError>),
```

Comment:
Not introduced by this PR, but it seems like it would be useful to print the `UnescapeError` content.